app/src/main/res/values/strings.xml

```diff
@@ -333,6 +333,8 @@
   <string name="app_version_name">App Version %s</string>
   <string name="app_last_update_date">The last update was installed on %s. Use the above version number to send feedback about bugs.</string>
   <!-- OptionsActivity -->
+  <string name="app_language_activity_title">App Language</string>
+  <string name="audio_language_activity_title">Default Audio Language</string>
   <string name="title_reading_text_size">Reading Text Size</string>
   <string name="title_app_language">App Language</string>
   <string name="title_default_audio">Default Audio Language</string>
```

Comment:
You can remover `title_app_language ` and `title_default_audio ` and whereever it was getting used just use `app_language_activity_title ` and `audio_language_activity_title `.

Reply:
Done